sdk/textanalytics/Azure.AI.TextAnalytics/src/AspectSentiment.cs

```diff
@@ -13,19 +13,19 @@ namespace Azure.AI.TextAnalytics
     /// For example in "The food at Hotel Foo is good", "food" is an aspect of
     /// "Hotel Foo".
     /// </summary>
-    public readonly struct AspectSentiment
+    public readonly struct OpinionTargetSentiment
     {
         private const double _neutralValue = 0d;
 
-        internal AspectSentiment(TextSentiment sentiment, string text, double positiveScore, double negativeScore, int offset)
+        internal OpinionTargetSentiment(TextSentiment sentiment, string text, double positiveScore, double negativeScore, int offset)
```

Comment:
I wonder if it's worth separating the opinion target and the sentiment about the opinion target into separate types?

Or, generally renaming this to OpinionTarget, and it can "have a" sentiment, rather than calling out at the top level that it "is a" sentiment.  (I realize this is in contradiction to an earlier suggestion I made, but now that I see it, I think it might be better to call it the target instead of target sentiment for clarity about what an opinion is.)

Reply:
That is a good point. Some people on the user study thought that `Sentiment` was used as a suffix to tell them that the type only contained the `Sentiment` which is not what we want.

Even in the current design, I think it is a good idea to remove it